internal/pkg/filesystem/knownpaths/paths_test.go

```diff
@@ -33,12 +33,12 @@ func TestKnownPathsIgnoredFile(t *testing.T) {
 	assert.NotNil(t, paths)
 	assert.NoError(t, err)
 	assert.Empty(t, paths.TrackedPaths())
-	assert.Equal(t, []string{`dir`}, paths.UntrackedPaths())
+	assert.Equal(t, []string{"description.md", "dir"}, paths.UntrackedPaths())
 
 	// Mark tracked some ignored file -> parent dir is marked as tracked
 	paths.MarkTracked("dir/.gitkeep")
 	assert.Equal(t, []string{`dir`}, paths.TrackedPaths())
-	assert.Empty(t, paths.UntrackedPaths())
+	assert.Equal(t, []string{"description.md"}, paths.UntrackedPaths())
```

Comment:
Pridal si subor asi do vsetkych `fixtures` adresarov, len napr. tu nedava zmysel.

Reply:
Testuje sa, ze v adresary je nejaky ignorovany subor, a ze sa ignoruje.

Reply:
Tam jsem to musel přidat, ten test jinak failoval.

Reply:
No ale padal preto, lebo si do adresara `ignored-file` pridal `description.md`, nie? :slightly_smiling_face: 
![image](https://user-images.githubusercontent.com/19371734/170653204-ff5be905-a253-40e7-aeb8-ba966ce1506a.png)